shared/config/happy_config.go

```diff
@@ -29,6 +29,7 @@ type Environment struct {
 	AutoRunMigrations  bool            `yaml:"auto_run_migrations"`
 	TaskLaunchType     util.LaunchType `yaml:"task_launch_type"`
 	LogGroupPrefix     string          `yaml:"log_group_prefix"`
+	StackOverrides     map[string]any  `yaml:"stack_overrides"`
```

Comment:
Any config values can be stored here to override the StackDefaults; and both are used to override module defaults.